lib/release_notes/changelog_parser.rb

```diff
@@ -42,7 +42,8 @@ def self.changelog_summary(prs)
 
     def self.changelog_prs(prs)
       return [] unless prs
-      prs.select { |pr| pr if pr[:text].include?(INCLUDE_PR_TEXT) }
+      prs.reject! { |pr| pr[:text].blank? } # Reject empty PRS texts
```

Comment:
Does this make any difference (performance or otherwise)?

Reply:
Nope. Unnecessary